api/api.go

```diff
@@ -0,0 +1,179 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"learningbay24.de/backend/config"
+	"learningbay24.de/backend/course"
+	"learningbay24.de/backend/models"
+)
+
+
+
+func GetCourseById(c *gin.Context) {
+    db := config.SetupDbHandle()
```

Comment:
Statt das Database jedes Mal zu erstellen lieber hier orientieren:
https://stackoverflow.com/questions/35672842/go-and-gin-passing-around-struct-for-database-context/60324692#60324692